region/lowernorfair/east.json

```diff
@@ -6420,6 +6420,28 @@
                 }
               ]
             },
+            {
+              "id": 3,
+              "strats": [
+                {
+                  "name": "Zebbo Farm",
+                  "notable": false,
+                  "requires": [
+                    {"or": [
+                      {"and": [
+                        "canPauseAbuse",
+                        "Grapple"
+                      ]},
+                      {"and": [
+                        "h_heatResistant",
+                        {"heatFrames": 50}
+                      ]}
+                    ]},
+                    {"refill": ["Energy", "Missile", "Super"]}
```

Comment:
`refill: missile` would need heatproof

Reply:
and `refill: Energy` is something like `canBeVeryPatient`.  I think you get like 20 health back per cycle if done really well.  But doing that increases the risk of mistakes and those almost like you 100.

Reply:
Good catch, removed that now. I'm planning on doing a separate PR for heatproof and/or non-respawning Norfair farms and can include the Missile one there (along with one to farm Power Bombs from the Violas)

Reply:
Just saw the comment about energy. Yeah, that's a good point. Maybe we could add a link from each farm to the opposite door node with no heat frames, as a (admittedly hacky) way to represent that you can farm just enough to get through the room without a net loss. And then we could include a canBeVeryPatient alternative that fully refills

Reply:
It does decently match how you try to go through the room while saving health.  Drop RNG becomes a potential problem though.  Usually only when not full on supers.

Reply:
Ok I took another pass at this. The assumption is that refilling on Supers & Power Bombs (but not energy) should be reasonable with Expert tech; it may take a while to do (and farm back up to the original energy level) but not to an extent requiring patience tech? Even with "bad" RNG (e.g. lots of Super drops quickly), you can still survive with health bomb and pause abuse, right?

Reply:
At health bomb its about every third farm cycle you gain enough health to get ammo drops again.  Pbs are still a rare drop from the bugs, but the Violas are right there.  At health bomb its hard to get any pbs from violas too.

Reply:
I was imaging the PB drops would come from resetting the room and killing Violas again, though you lose health with that so it would be slow. Do you think Supers & PB refill should just be removed here (or only included with canBeVeryPatient)?